test/runtests.jl

```diff
@@ -32,19 +32,14 @@ H_GT = [6 10 8 2 0 1 2 10;
      4 9 10 7 7 0 0 0
     ]
 
-@testset "test top wrapper" begin
+@testset "test top wrapper" begin        
```

Comment:
Extra whitespace, remove. (If you like, in vscode you can change a setting so that all trailing white space is deleted.)

Reply:
I added a test

Reply:
"Extra whitespace, remove" means the following: in this PR, you're accidentally inserting a tab or series of spaces after the `begin` on the same line. I'm suggesting you delete it, then there won't be a change on this line. And since this wasn't clear I looked up a link for you: https://bobbyhadz.com/blog/remove-trailing-whitespace-vscode

GitHub is highlighting this for you because many projects (Julia itself being one of them) test for trailing whitespace and cause CI to fail if it is detected.